### PR TITLE
Tree best parser

### PR DESCRIPTION
**Pull request by anilthanki**

### CODE REVIEW

**Comment by nsoranzo** [comment]
Third-party code should not be included, e.g. [Bio::Phylo](http://search.cpan.org/~rvosa/Bio-Phylo/).

**Comment by nsoranzo** [comment]
Remove .DS_Store files.

**Comment by bgruening** [comment]
I'm wondering how this can happen if the gitignore file is up-to-date. 
@anilthanki you can also add more to the gitignore file: https://github.com/galaxyproject/tools-iuc/blob/master/.gitignore

**Comment by georgek** [comment]
If you use software that writes files like .DS_Store that you probably never want to commit to any repository ever, you should put these in your own global gitignore. Make a ~/.gitignore with .DS_Store in it etc. and do `git config --global core.excludesfile ~/.gitignore`. The project's own .gitignore file should be for things generated by this project (like compiler output etc.), not things created by your personal tools. Otherwise the .gitignore could grow massive if everyone uses different tools.

File: tools/TreeBestParser/TreeBest_parser.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        $inputFile > $output
+        ]]>
+    </command>
+
+    <inputs>
+        <param name="inputFile" type="data" format="nhx" label="Newick file" help="Newick file format" />
+    </inputs>
+
+    <outputs>
+         <data name="output" format="txt" label="$tool.name on ${on_string}" />
```
Comment (nsoranzo) [inline tools/TreeBestParser/TreeBest_parser.xml:18]: I guess `format` should be "json" or an Aequatus-specific JSON subtype.

**Pull request closed without merging**